Adding support for the sama5d3_ stephan board

julikarst opened the pull request:
Adding support for the in-house custom sama5d3 board of the Fritz Stephan company (http://www.stephan-gmbh.com/).
We are a german company building respirators, thus we won't sell the board itself but the devices using it.
We plan to use the board in several devices for the next years. We would like to go mainline for that.

Extracted from our fork:
https://github.com/julikarst/at91bootstrap

julikarst commented:
There are mainly differences in the RAM timings, but anyway we will stay with our fork. Thank you for reviewing.

ehristev commented:
We can surely merge your request if you provide us with the information and we find the appropriate differences. It was merely a question of what is different as I can only see two different numbers w.r.t.  ddramc_config->rtr and AT91C_DDRC2_TRPA_(2)
Is there anything different beside these two values ? What is the impact ? Do you have a different DDR module? Is there any schematic available for your contrib board ?

julikarst commented:
No, these two numbers are the only differences. Yes, we use a different DDR module. Sadly we cannot publish any shematics of our boards.

ehristev commented:
Can you fixup the file headers. Put your own header copyright and below write that you base your file on the acqua one. And that the difference is in DDR timings.
Then, also rebase your commits on the latest master branch. 
Fixup your commit shortlog : board: contrib: etc.
(Make it look like the commits that added the other boards).
Add your signed-off-by tag (git commit -s)
Thanks !

ehristev commented:
Something is wrong with your pull request , you appear to have included more commits than your own. Please rebase or do another pull request with just your commits, and fix the comments requested earlier.
Thanks !

julikarst commented:
Yes indeed. Something went wrong with the rebase. I'll do another clean pull request tomorrow. Sorry for the circumstances!

julikarst commented:
The rebase was clean this time. I fixed the commit log and added signed-off-by tag. I'll do the requested changes in the copyright section tomorrow. Thanks for your patience (I'm not familiar with git commands like rebase :D )
